src/Microsoft.AspNetCore.Mvc.Razor/Internal/ViewLocationCacheKey.cs

```diff
@@ -67,6 +71,11 @@ public struct ViewLocationCacheKey : IEquatable<ViewLocationCacheKey>
         public string AreaName { get; }
 
         /// <summary>
+        /// Gets the page name.
+        /// </summary>
+        public string PageName { get; }
```

Comment:
I added some comments here. I don't think the naming would make it better, its just as ambiguous as before.